src/app/visual/infinite-scroll/infinite-scroll-visual.component.ts

```diff
@@ -6,4 +6,27 @@ import {
   selector: 'infinite-scroll-visual',
   templateUrl: './infinite-scroll-visual.component.html'
 })
-export class InfiniteScrollVisualComponent { }
+export class InfiniteScrollVisualComponent {
+  public firstList: any[] = [
+    0, 1
+  ];
+  public secondList: any[] = [
+    0, 1
+  ];
+
+  public secondEnabled = true;
+
+  public addToFirst() {
+    setTimeout(() => {
```

Comment:
When I click the "Load more" buttons, I see an endless wait component and the list is not updated?

Also, will the `setTimeout` affect the screenshots?

Reply:
Sorry, I took those back out because the visual test is taking a wait-state screenshot. So these need to start loading and never stop to prevent a race condition with that screenshot. So the two methods are not tied to the template. I left them in to use for checking things when we serve locally. Should I just remove them?

Reply:
Gotcha. Is there a way to make it easy for people to manually test, and to allow the appropriate screenshots to be taken?

Reply:
I guess I could add another infinite scroll and change the target. So one would load new things, but the other would not. And the screenshot could use the one that does not load anything

Reply:
done